datacontract/export/dbt_converter.py

```diff
@@ -239,9 +246,21 @@ def _to_column(field_name: str, field: Field, supports_constraints: bool, adapte
                     }
                 }
             )
+    if field.references is not None:
+        ref_source_name = data_contract_spec.id
+        table_name, column_name = get_table_name_and_column_name(field.references)
+        if table_name is not None and column_name is not None:
+            column["data_tests"].append(
+                {
+                    "relationships": {
+                        "to": f"""source("{ref_source_name}", "{table_name}")""",
+                        "field": f"{column_name}",
+                    }
+                }
+            )
 
-        if not column["data_tests"]:
```

Comment:
I made this correction, but the scope in which this processing was performed was incorrect 🙏 

- https://github.com/datacontract/datacontract-cli/pull/548/files#diff-b03f6e792b001775280d2 e272e3faef45c6880886f91c19597a23662b7f574a3R242-R244